libs/klv/src/main/java/org/codice/alliance/libs/klv/GeometryUtility.java

```diff
@@ -121,7 +121,17 @@ public static String attributeToLineString(
 
     Geometry geometry = convertCoordinatesToGeometry(coordinates);
 
-    return convertGeometryToWkt(geometryOperator.apply(geometry, geometryOperatorContext));
+    if (geometry == null) {
+      return null;
+    }
+
+    geometry = geometryOperator.apply(geometry, geometryOperatorContext);
```

Comment:
Can geometryOperator.apply not handle a null geometry? Would allow you to remove your first `if = null` check if it does.

Reply:
The GeometryOperator contract does not specify how nulls are handled. I'd rather play it safe.